core-impl/proxy/pom.xml

```diff
@@ -39,11 +42,24 @@
             <artifactId>slf4j-log4j12</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
Can merge the latest master code. The dependency related to jmh is already added in `pom.xml`.